websites/I/Incels.is/presence.ts

```diff
@@ -0,0 +1,129 @@
+const presence = new Presence({
+		clientId: "1094962047906758796",
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+
+const enum Assets {
+	Logo = "https://incels.is/logo/512x512.png",
+}
+
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+			largeImageKey: Assets.Logo,
+			startTimestamp: browsingTimestamp,
+		},
+		threadTitleElement = document.querySelectorAll(
+			".p-title-value"
+		)[0] as HTMLHeadingElement,
+		threadTitle: string | undefined = threadTitleElement
+			? threadTitleElement.textContent
+			: // eslint-disable-next-line no-undefined
+			  undefined,
+		threadTag =
+			document.querySelector('[class*="label label--custom-"]').textContent ??
+			// eslint-disable-next-line no-undefined
+			undefined,
```

Comment:
Don't disable eslint rules without good reason. This presence might not require explicit `undefined` types to work

Reply:
If the element happens not to exist on the current page the user is on, the console will periodically spit out errors as it's trying to access the `textContent` of something that is not in the DOM. This was my fix for that issue (well, mine and GitHub Copilot's).

If you know another way of avoiding this from happening, please tell me about it and I'll implement it.

Reply:
`?.textContent` shouldn't it be that?

Reply:
The IDE I use at work usually tells me whenever a variable might be undefined, so I'm blaming VSCode for this one lmao.

But yeah, implemented it.